### PR TITLE
Update package.json and CHANGELOG for release 4.0.0

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 4.0.0 (2020-10-09)
+- Added the option to set column picker sorting to `None` to allow column options to render in the order provided. [#12](https://github.com/blackbaud/skyux-data-manager/pull/12)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:2]:
```suggestion
- Added an option to disable column picker sorting to render column options in the order provided. [#12](https://github.com/blackbaud/skyux-data-manager/pull/12)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+- Updated the data manager component to scroll to the top of the data view when the active view is changed. [#15](https://github.com/blackbaud/skyux-data-manager/pull/15)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Updated the data manager component to scroll to the top of the data view when the active view changes. [#15](https://github.com/blackbaud/skyux-data-manager/pull/15)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+- Updated the data manager service to log a warning if `initDataManager` or `initDataView` is called for a data manager or view that has already been initialized. [#14](https://github.com/blackbaud/skyux-data-manager/pull/14)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:4]:
```suggestion
- Updated the data manager service to log a warning if `initDataManager` or `initDataView` are called after a data manager or view is initialized. [#14](https://github.com/blackbaud/skyux-data-manager/pull/14)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.0](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.0)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+- Added an option to `getDataStateUpdates` to only receive updates if one of a provided list of properties changes, or if a provided comparator indicates changes. [#16](https://github.com/blackbaud/skyux-data-manager/pull/16)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:5]:
```suggestion
- Added a `getDataStateUpdates` option to only receive updates when provided properties change or when a provided comparator indicates changes. [#16](https://github.com/blackbaud/skyux-data-manager/pull/16)
```

**Review by blackbaud-johnly — DISMISSED** [review]
A few style/wording suggestions, but feel free to ignore if you don't think the changes are worth making.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.0](https://developer.blackbaud.com/skyux-data-manager?_sp.spa=pr-release-4.0.0)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**